docs/packages/markdown/parseMarkdown.js

```diff
@@ -414,14 +414,18 @@ function prepareMarkdown(config) {
         contents.push(`
 ## Unstyled
 
-The component also comes with an [unstyled version](${headers.unstyled}). It's ideal for doing heavy customizations and minimizing bundle size.
+:::success
```

Comment:
it's unclear to me that it should be an alert. I think that the least often we use them, the more they mean something.

Reply:
I don't have a strong feeling about it either way. The reasoning was that we've started to use the `:::success` callout as a way to alert the reader about helpful hints or useful patterns/practices, and I felt that the info here fits that description. I also like that it draws more attention to the existence of Base (for educational purposes). But I think it would be fine without the callout, too.